src/main/scala/com/danielasfregola/twitter4s/http/oauth/OAuth1Provider.scala

```diff
@@ -30,7 +30,7 @@ private[twitter4s] class OAuth1Provider(consumerToken: ConsumerToken, accessToke
     val params = basicOAuth2Params(callback)
     for {
       signature <- oauth1Signature(params)
-    } yield (params + signature).mapValues(_.urlEncoded)
+    } yield (params + signature).mapValues(_.urlEncoded).toMap
```

Comment:
`Map.mapValues()` now returns a `MapView` rather than a `Map` - we usually want a `Map` though, so we can just add in a `.toMap` to the end of the expression - it's harmless in Scala 2.12, and allows the code to compile in Scala 2.13. 

`Map.mapValues()` itself is deprecated in Scala 2.13, but using the new recommended alternative doesn't work natively in Scala 2.12.

https://docs.scala-lang.org/overviews/core/collections-migration-213.html#what-are-the-breaking-changes

See also scala/bug#10919&  scala/scala#7014.